### PR TITLE
Fixes icon positioning in IE11 #1588

### PR DESCRIPTION
**Pull request by toni-leigh**
# Description

Fixes icon positioning in IE11 #1588

# TODO
- [x] Release notes

### CODE REVIEW

**Comment by andrewjtait** [comment]
This will fix the issue for `Message`, but not for `Toast` - right?

(thats fine btw - but may need to keep the issue open)

**Comment by toni-leigh** [comment]
OK, cool, I think I can put that fix in too today

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,10 @@
 
 * `SettingRow` - the left hand segment now has a `max-width` of 300px.
 
+## Bug Fixes
+
+* `Message`: Fixes the icon positiing in IE11
```
Comment (andrewjtait) [inline CHANGELOG.md]: Add a bullet point for `Toast` too please. Add move to `2.6.1`.

**Comment by toni-leigh** [comment]
Now fixed @andrewjtait

**Comment by andrewjtait** [comment]
Tested in IE11 and modern browsers ✅

**Pull request merged**